invoice/services/bill.py

```diff
@@ -0,0 +1,59 @@
+from typing import Dict, Union, List
+
+from invoice.models import Bill, BillItem
+from invoice.services.base import BaseService
+from invoice.services.service_utils import _get_generic_type
+from invoice.validation.bill import BillModelValidation, BillItemStatus
+from core.signals import *
+
+
+class BillService(BaseService):
+
+    OBJECT_TYPE = Bill
+
+    def __init__(self, user, validation_class: BillModelValidation = BillModelValidation):
+        super().__init__(user, validation_class)
+        self.validation_class = validation_class
+
+    def _base_payload_adjust(self, bill_data):
+        return self._evaluate_generic_types(bill_data)
+
+    def bill_validate_items(self, bill: Bill):
+        # TODO: Implement after calculation rules available
```

Comment:
Is it still to do?

Reply:
I think when this 1st rule regarding creating Bill is ready, then we can implement this validation.